packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -420,6 +432,27 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       return null;
     }
 
+    if (this.disabled) {
+      return html`
+        <div class="${classMap(inputClasses)}">
+          <input
+            id="people-picker-input"
+            class="search-box__input"
+            type="text"
+            placeholder=${placeholder}
+            label="people-picker-input"
+            aria-label="people-picker-input"
+            role="input"
+            @keydown="${this.onUserKeyDown}"
+            @keyup="${this.onUserKeyUp}"
+            @blur=${this.lostFocus}
+            @click=${this.handleFlyout}
+            disabled
+          />
+        </div>
+      `;
+    }
+
```

Comment:
Lit-html allows you to specify a boolean attribute and it will only be set when it's true. Remove this duplicate code and update the original code below by adding the `disabled` check like so:

```diff
...
   @click=${this.handleFlyout}
+  ?disabled=${this.disabled}
/>
```

Reply:
This is awesome!